scene/theme/theme_db.cpp

```diff
@@ -435,6 +442,7 @@ ThemeDB *ThemeDB::get_singleton() {
 
 ThemeDB::ThemeDB() {
 	singleton = this;
+	MessageQueue::get_singleton()->push_callable(callable_mp(this, &ThemeDB::_sort_theme_items));
```

Comment:
Looks like this line causes a crash in unit tests on CI on every platform.

Reply:
Are you trying to call it just once for all registration calls by delaying it like that? If so, I'm not sure it's going to be reliable, but we can give it a go, if the crash is fixable.

Reply:
For better reliability I could add `queue_sort_items()` to call when an item is added, but currently adding is done only once, during initialization, so it's not needed.